nextflow.config

```
@@ -270,14 +279,17 @@ profiles {
         podman.enabled         = false
         shifter.enabled        = false
         charliecloud.enabled   = false
+        apptainer.enabled      = false
     }
     podman {
         podman.enabled         = true
+        podman.registry        = 'quay.io'
```

```suggestion
        // podman.registry        = 'quay.io'
```
Let's keep it to the old default until non-quay modules are updated (i.e. until this [#3344](https://github.com/nf-core/modules/pull/3344) PR is merged).